go.mod

```diff
@@ -14,20 +14,34 @@ require (
 	google.golang.org/protobuf v1.28.1
 	gorm.io/driver/postgres v1.4.7
 	gorm.io/gorm v1.24.5
+	k8s.io/apimachinery v0.27.2
```

Comment:
@jeyhun why do you need k8s ? please remove the dependency

Reply:
I was using an error struct from k8s' package, since I wanted my unit test to use not just the `DbError` struct for errors. I've removed the k8s dependency and started using the built-in error package to generate an error that's not `DbError`.